test/compile/data/source.test.ts

```diff
@@ -20,9 +20,10 @@ describe('compile/data/source', function() {
         assert.deepEqual(source.data.values, [{a: 1, b:2, c:3}, {a: 4, b:5, c:6}]);
       });
 
-      it('should have no source.format.type', function(){
-        assert.deepEqual(source.data.format, undefined);
-      });
+      // https://github.com/vega/vega-parser/pull/60
```

Comment:
So this can be revert after https://github.com/vega/vega-parser/pull/60 is included in the next version of Vega?

Reply:
Probably, yes